# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: xpcom/ds/StaticAtoms.py
# Commit: 72e19189bb99
# Full Hash: 72e19189bb99e159b2974cbcda69647211962df3
# Author: Brendan Dahl <bdahl@mozilla.com>
# Date: 2019-06-14 03:39:39
# Regressor Bug: 1552049
# File Overlap Count: 1
# Description:
#   Bug 1552049 - Enable XUL persistence for all system privilege docs. r=smaug
#   
#   Remove the mozpersist attribute and XULDocument requirement to use XUL
#   persistence and allow any system privilege document to use it by default.
#   
# ==============================================================================

diff -r bc46a287b898 -r 72e19189bb99 xpcom/ds/StaticAtoms.py
--- a/xpcom/ds/StaticAtoms.py	Thu Jun 13 17:37:15 2019 +0000
+++ b/xpcom/ds/StaticAtoms.py	Thu Jun 13 18:15:54 2019 +0000
@@ -42,7 +42,6 @@
     Atom("moztype", "_moz-type"),
     Atom("mozdirty", "_moz_dirty"),
     Atom("mozdisallowselectionprint", "mozdisallowselectionprint"),
-    Atom("mozpersist", "mozpersist"),
     Atom("mozdonotsend", "moz-do-not-send"),
     Atom("mozeditorbogusnode", "_moz_editor_bogus_node"),
     Atom("mozgeneratedcontentbefore", "_moz_generated_content_before"),
